gke-deploy/core/resource/resource.go

```diff
@@ -159,8 +161,13 @@ func SaveAsConfigs(ctx context.Context, objs Objects, outputDir string, lineComm
 	if err := oss.MkdirAll(ctx, outputDir, os.ModePerm); err != nil {
 		return fmt.Errorf("failed to create output directory %q: %v", outputDir, err)
 	}
-	for baseName, obj := range objs {
-		filename := filepath.Join(outputDir, baseName)
+
+	aggregateName := "aggregate.yaml"
```

Comment:
Mentioned in person - maybe change this to "aggregated-resources.yaml"? If you choose to do so, make sure to fix up the tests that reference this as well. Might make sense to put it in a variable name that can be reference by the unit tests.

Reply:
@demobox FYI

Reply:
> maybe change this to "aggregated-resources.yaml"

+1 to something like that. Or `kubernetes-resources.yaml`, `kubernetes-config.yaml`, `k8s-config.yaml` or so?